openedx/core/djangoapps/user_authn/config/waffle.py

```diff
@@ -37,3 +37,17 @@
     'enable_pwned_password_api',
     __name__
 )
+
+
+# .. toggle_name: REDIRECT_TO_LMS
```

Comment:
Name should indicate that this only applies to admin -- maybe `ADMIN_AUTH_REDIRECT_TO_LMS` or similar?

Reply:
Updated the toggle switch name to the suggested one.